### DISCUSSION TITLE
Actionbar Buttons not Showing

### DISCUSSION THREAD

**Discussion by ghost** [SFX-WoW/Masque#181] [Issues] [answered]
The combination of using ElvUI_ExtraActionBars + Masque results in ElvUI_ExtraActionBars not showing all buttons upon login/reload. ElvUI_ExtraActionBars has no problems whatsoever whenever Masque is disabled it only occurs when Masque is enabled. Brought this issue up to the developer of ElvUI_ExtraActionBars however he said the issue lies in Masque. Could this me looked into please.

**Comment by StormFX** [comment]
First, make sure you're not using the "AddonSkins" add-on/module. It doesn't work well with Masque, though I've heard there's a Masque-specific option in there. I'm not sure on that though, because I don't use ElvUI.

Secondly, Masque does not hide entire buttons, it only affects their child elements. So if some buttons are completely disappearing, it's not Masque doing it.

**Comment by ghost** [comment]
As ive said the only addons enabled are ElvUI, ElvUI_ExtraActionBars & Masque. In which case half the buttons of bar7/8/9/10 which are all part of ElvUI_ExtraActionBars do not show up. As soon as Masque is disabled in the addons pane or Masque has been disabled in ElvUI itself the buttons show up again.

**Comment by StormFX** [comment, marked as answer]
I understand that. But as I said, Masque is effectively a library. If using it with ExtraActionBars isn't working as expected, it's a problem with the implementation by ExtraActionBars. This is easily demonstrated by the fact that it works fine with the rest of ElvUI and any number of other action bar add-ons.

**Comment by ghost** [comment]
Fair enough just trying to figure out with both of you guys what the problem might be.
